engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -382,11 +382,18 @@ object Main {
     mainModule: Module,
     rootPkgPath: Option[File],
     mainMethodName: String = "main"
-  ): Value = {
+  ): Unit = {
     val mainCons = mainModule.getAssociatedConstructor
     val mainFun  = mainModule.getMethod(mainCons, mainMethodName)
     try {
-      mainFun.execute(mainCons.newInstance())
+      mainFun match {
+        case Some(main) => main.execute(mainCons.newInstance())
+        case None =>
+          err.println(
+            "Your module does not contain a `main` function. It could not " +
```

Comment:
This message could be improved: `Your module` doesn't mean anything if I'm running a package. You probably should include the module name instead.